### PR TITLE
BLUI-2965 - Cleanup: Fix scroll width on registration workflow and change password success screen

### LINKED ISSUE #75: Scroll containers don't fill the screen
#### Describe the bug
The ScrollViews on many screens are nested within a View with a max width of 600, which means you can only scroll the screen by dragging your finger in the middle 600px wide column of the screen.

#### What is the expected behavior?
The page should be scrollable by dragging the finger anywhere on the page.
	
#### Anything else to add?
We should be able to let the ScrollView fill the entire page and set a max width on its contents instead.

### PR DESCRIPTION
**Pull request by daileytj**
Fixes #75 & BLUI-2965.

#### Changes proposed in this Pull Request:
- Make scroll containers fill the screen for registration workflow email screen and the change password success screen

#### To Test:
- `git clone git@github.com:brightlayer-ui/react-native-workflows.git -b feature/blui-2965-fix-scroll-container`
- `yarn start:example-ipad`
- If the ipad script doesn't work update the example script for ipad to suit your local environment needs
- check self registration workflow email screen and the change password success screen and ensure that you can scroll from anywhere, not just the inner 600px of content

### CODE REVIEW

**Comment by ektasawant** [comment]
forgot_password
Able to scroll forgot password screen but not success screen is it correct behaviour ?

File: login-workflow/example/App.tsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import 'react-native-gesture-handler';
 import React from 'react';
 import { Provider as ThemeProvider } from 'react-native-paper';
-import { SafeAreaProvider, useSafeAreaInsets } from 'react-native-safe-area-context';
-import { ThemedButton as Button } from '@brightlayer-ui/react-native-components/themed';
+import { SafeAreaProvider /*, useSafeAreaInsets */ } from 'react-native-safe-area-context';
+// import { ThemedButton as Button } from '@brightlayer-ui/react-native-components/themed';
```
Comment (ektasawant) [inline login-workflow/example/App.tsx:5]: Any reason to keep commented code?

Comment (daileytj) [reply login-workflow/example/App.tsx:5]: The example app has a lot of uncommented code. It is left in there to quickly uncomment things to test various features. This is an internal project that we use primarily for testing the react-native-auth-workflow package.

**Comment by daileytj** [comment]
@ektasawant That view isn't scrollable at any screensize so this is working as expected if all of the scroll views are scrolling at full width.

**Comment by jeffvg** [comment]
all seems good, even the success screen now


https://user-images.githubusercontent.com/37150565/188178773-1ec0e391-0fc1-4b84-8a46-5eb62a31be05.mov

**Review by jeffvg — COMMENTED** [review]
looks good. Do you need change log entry for this

**Review by jeffvg — APPROVED** [review]

**Pull request merged**